Dockerfile

```
@@ -8,25 +8,36 @@ RUN --mount=source=.,target=/mnt,type=bind \
                                                vim python3-all pciutils binutils tcpdump \
                                                rsyslog iptables ssl-cert openssl ca-certificates \
                                                curl nmap hydra hping3 iperf3 d-itg socat \
- && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb \
- && echo "root:hackinsdn" | chpasswd \
+ && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb
+
+RUN rm -rf /var/lib/apt/lists/*
```

Let's try to group steps that have a strong relationship together. I suggest the following:
```suggestion
 && apt install -y /mnt/misc/shellinabox_2.21.1_amd64.deb \
 && rm -rf /var/lib/apt/lists/*
```